admin/client/ragflow_client.py

```
@@ -495,113 +561,146 @@ def set_variable(self, command):
+            if res_json["code"] == 0:
+                self._print_table_simple(res_json["data"])
+            else:
+                print(f"Fail to show fingerprint, code: {res_json['code']}, message: {res_json['message']}")
         else:
-            print(f"Fail to show fingerprint, code: {res_json['code']}, message: {res_json['message']}")
+            print(f"Fail to show fingerprint, code: {response.status_code}, body: {response.text}")
 
     def set_license(self, command):
         if self.server_type != "admin":
             print("This command is only allowed in ADMIN mode")
         license = command["license"]
```

Ruff (0.15.21)

[error] Variable `license` is shadowing a Python builtin

(A001)